x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -216,12 +216,20 @@ export async function getInstalledPackages(options: GetInstalledPackagesOptions)
   const integrationsWithManifestContent = integrations.map((integration) => {
     const { name, version } = integration;
     const integrationAsset = integrationManifests.get(`${name}-${version}/manifest.yml`);
-
     return {
       ...integration,
       title: integrationAsset?.title ?? undefined,
       description: integrationAsset?.description ?? undefined,
       icons: integrationAsset?.icons ?? undefined,
+      dataStreams: integration.dataStreams.map((dataStream) => {
```

Comment:
I'm wondering if we need to read the manifests, there is an open pr where the package installation will add the data stream display name to the saved object, could we use that? https://github.com/elastic/kibana/pull/180684/files#diff-cf9cec44de2ad0a6a3b74cca05e5308231d57d5c3e180ae4bc5114c2bf9af4eb